lib/assessment.js

```diff
@@ -5,6 +5,12 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  var result = argument[0];
+    for(var i =1; i < argument.legth; i++) {
+      if(argument[i] < result) {
+        resutl = arguement[i];
+      }
+    }
```

Comment:
This function is good in most of the ways that matter most, but it needs some fixing. Most important thing to get out of this is that the way to access arguments inside a function is the `arguments` pseudo-array. It's called `arguments`, plural, because it's expected to have more than one. I'll only count that mistake once since you did this all in one session. 3/5